aaelf32/aaelf32.rst

```diff
@@ -0,0 +1,3289 @@
+
+PT_ARM_EXIDX (alias PT_ARM_UNWIND) describes the location of a program’s unwind
+tables.
+
+.. rubric:: p_flags
+
+There are no processor-specific flags.  All bits in the PT_MASKPROC part of
+this field are reserved to future revisions of this specification.
+
+Platform architecture compatibility data
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This data describes the platform capabilities required by an executable file.
+It can be constructed by a linker using the attributes [Addenda32_] found in
+its input relocatable files, or otherwise.
+
+If this segment is present it shall contain at least one 32-bit word with
+meaning defined by `Common architecture compatibility data masks`_,
+`Architecture compatibility data formats`_, `Architecture profile compatibility
+data`_, and `Architecture version compatibility data`_ below.
+
+.. _Common architecture compatibility data masks:
+
+.. table:: Common architecture compatibility data masks
+  :widths: 35 15 50
+
+  +------------------------+------------+---------------------------------------------------------------------------------------------+
+  | Name                   | Value      | Meaning                                                                                     |
+  +========================+============+=============================================================================================+
+  | PT_ARM_ARCHEXT_FMTMSK  | 0xff000000 | Masks bits describing the format of data in subsequent words. The masked value is           |
+  |                        |            | described in `Architecture compatibility data formats`_, below.                             |
+  +------------------------+------------+---------------------------------------------------------------------------------------------+
+  | PT_ARM_ARCHEXT_PROFMSK | 0x00ff0000 | Masks bits describing the architecture profile required by the executable. The masked value |
+  |                        |            | is described in `Architecture profile compatibility data`_, below.                          |
+  +------------------------+------------+---------------------------------------------------------------------------------------------+
+  | PT_ARM_ARCHEXT_ARCHMSK | 0x000000ff | Masks bits describing the base architecture required by the executable. The masked value is |
+  |                        |            | described in `Architecture version compatibility data`_, below.                             |
+  +------------------------+------------+---------------------------------------------------------------------------------------------+
+
+The following table lists the architecture compatibility data formats defined
+by this ABI. All other format identifiers are reserved to future revisions of
+this specification.
+
+.. _Architecture compatibility data formats:
+
+.. table:: Architecture compatibility data formats
+  :widths: 35 15 50
```

Comment:
as described above on the version history table, please use a class to specify the width